tests/compatibility_tests/create_structured_index/test_create_structured_index.py

```diff
@@ -0,0 +1,66 @@
+import pytest
+
+from tests.compatibility_tests.base_test_case.base_compatibility_test import BaseCompatibilityTestCase
+
+@pytest.mark.marqo_version('2.0.0')
+class TestCreateStructuredIndex(BaseCompatibilityTestCase):
+    indexes_settings_to_test_on = [
+        {
+            "type": "structured",
```

Comment:
There are field types that 2.0 didn't support. You need to cover those

Reply:
Have covered them in tests corresponding to 2.2, 2.9, 2.12